.travis.yml

```diff
@@ -20,7 +26,7 @@ r_github_packages:
   - USGS-R/rloadest@v0.4.2
 
 after_success:
-  - Rscript -e 'covr::codecov(); covr::coveralls()'
+  - Rscript -e 'library(covr);covr::codecov(); covr::coveralls()'
```

Comment:
ok, sounds good. yeah, just do another commit - it automatically gets attached to this pull request as long as the pull request remains open. it's fine/good with me if you make the commit to this file now - no need to wait until i'm done with the whole review.